app/serializers/initial_state_serializer.rb

```diff
@@ -10,7 +10,7 @@ class InitialStateSerializer < ActiveModel::Serializer
   attribute :critical_updates_pending, if: -> { object&.role&.can?(:view_devops) && SoftwareUpdate.check_enabled? }
 
   has_one :push_subscription, serializer: REST::WebPushSubscriptionSerializer
-  has_one :role, serializer: REST::RoleSerializer
+  has_one :role, serializer: REST::CredentialRoleSerializer
```

Comment:
For initial state for the current user, we want to include the permissions (since it's related to the current user)